lms/static/sass/views/_verification.scss

```diff
@@ -821,6 +829,114 @@
 
     // indiv slides - review
     #wrapper-review {
+      color: $black;
+
+      .page-title {
+          border-bottom: 2px solid $darkGrey;
+          padding-bottom: 15px;
```

Comment:
For all margin / padding rules, especially when using 5px, 10px, 15px, 20px, 30px, 40px (or larger multiples of 20px) use the $baseline variable. I won't comment on each instance in this PR, but there are a number of cases where I would update to use $baseline, ($baseline/2), etc. 

In addition, for padding left /right (or any rule where there would be an RTL impact, use @include padding(n, n, n, n);